src/adminPage/adminPage.component.tsx

```diff
@@ -67,214 +62,113 @@ const styles = (theme: Theme): StyleRules =>
-              labelPlacement="end"
-            />
-            <Button
-              style={{ float: 'right' }}
-              variant="contained"
-              color="primary"
-              onClick={() =>
-                setScheduledMaintenanceState(tempScheduledMaintenance)
-              }
-            >
-              <Typography color="inherit" noWrap style={{ marginTop: 3 }}>
-                {getString(props.res, 'save-button')}
-              </Typography>
-            </Button>
-          </div>
-        </Paper>
-        <Paper className={props.classes.paper}>
-          <Typography variant="h4">
-            {getString(props.res, 'maintenance-title')}
-          </Typography>
-          <TextareaAutosize
-            className={props.classes.textArea}
-            aria-label={getString(props.res, 'maintenance-message-arialabel')}
-            rows={7}
-            placeholder={getString(props.res, 'message-placeholder')}
-            value={tempMaintenance.message}
-            onChange={(e) =>
-              setTempMaintenance({
-                ...tempMaintenance,
-                message: e.currentTarget.value,
-              })
-            }
-          />
-          <div style={{ display: 'row' }}>
-            <FormControlLabel
-              style={{ float: 'left' }}
-              value={tempMaintenance.show}
-              control={
-                <Checkbox
-                  checked={tempMaintenance.show}
-                  onChange={(e) =>
-                    setTempMaintenance({
-                      ...tempMaintenance,
-                      show: e.target.checked,
-                    })
-                  }
-                  inputProps={{
-                    'aria-label': getString(
-                      props.res,
-                      'maintenance-checkbox-arialabel'
-                    ),
-                  }}
-                  color="secondary"
-                />
-              }
-              label={getString(props.res, 'display-checkbox')}
-              labelPlacement="end"
-            />
-            <Button
-              style={{ float: 'right' }}
-              variant="contained"
-              color="primary"
-              onClick={() => setMaintenanceState(tempMaintenance)}
-            >
-              <Typography color="inherit" noWrap style={{ marginTop: 3 }}>
-                {getString(props.res, 'save-button')}
-              </Typography>
-            </Button>
+      <Switch>
+        <Route exact path={adminRoutes.maintenance}>
+          <div
+            id="maintenance-panel"
+            aria-labelledby="maintenance-tab"
+            role="tabpanel"
+            hidden={tabValue !== 'maintenance'}
+          >
+            <MaintenancePage />
           </div>
-        </Paper>
-      </div>
+        </Route>
+
+        {Object.entries(pluginRoutes).map(([key, value]) => {
+          const pluginId = key.endsWith('_admin') ? key.slice(0, -6) : key;
```

Comment:
`getPluginRoutes` no longer appends `_admin` to admin routes so this logic could be simplified to just use `key`